cfme/tests/ansible/test_embedded_ansible_manual.py

```diff
@@ -0,0 +1,1090 @@
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: medium
+        initialEstimate: 1h
+        tags: ansible_embed
+    """
+    pass
+
+
+@pytest.mark.tier(1)
+def test_embed_tower_dashboard():
+    """
+    Check dashboard view has been added to existing Tower provider screens
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: medium
+        initialEstimate: 1/6h
+        tags: ansible_embed
+    """
+    pass
+
+
+@pytest.mark.tier(3)
+def test_embed_tower_retire_service_with_instances_ec2():
+    """
+    Retire Service+instances which were deployed by playbook from CFME UI.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: medium
+        initialEstimate: 1h
+        tags: ansible_embed
+    """
+    pass
+
+
+@pytest.mark.tier(3)
+def test_embed_tower_playbook_links():
+    """
+    There are links to repo"s within the playbook table. Clicking in this
+    cell will navigate to the details of the Repo.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: medium
+        initialEstimate: 1/6h
+        tags: ansible_embed
+    """
+    pass
+
+
+@pytest.mark.tier(3)
+def test_embed_tower_exec_play_against_machine_multi_appliance():
+    """
+    User/Admin is able to execute playbook without creating Job Temaplate
+    and can execute it against machine with machine credentials. Deploy 2
+    appliances, second one as unconfigured, through appliance_console join
+    the
+    region of first appliance. Enable embedded ansible on 2nd appliance.
+    From first appliance, add scm, credentials, new catalog, catalog item
+    of AnsiblePlaybook type. Select playbook e.g. dump_all_vars and order
+    it. When asked what machine to run it against, pick any rhel7 machine.
+    Playbook should be executed successfully.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: medium
+        initialEstimate: 1h
+        tags: ansible_embed
+    """
+    pass
+
+
+@pytest.mark.tier(3)
+def test_embed_tower_playbooks():
+    """
+    playbooks included under ansible shown in a table view (automation-
+    ansible-playbooks)
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: medium
+        initialEstimate: 1/6h
+        tags: ansible_embed
+    """
+    pass
+
+
+@pytest.mark.tier(1)
+def test_embed_tower_failover():
+    """
+    Check that ansible fails over to new region correctly
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: medium
+        initialEstimate: 1h
+        tags: ansible_embed
+    """
+    pass
+
+
+@pytest.mark.tier(1)
+def test_embed_tower_add_scm_credentials():
+    """
+    Add SCM credentials for private GIT repo.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
```

Comment:
missing `caseimportance` here

Reply:
If nothing specified it will assume to be High, right .?

Reply:
@sbulage you are correct, ignore me